src/server/rpc/procedures/google-maps/google-maps.js

```diff
@@ -206,9 +206,39 @@ StaticMap.prototype.getLatitude = function(y){
     });
 };
 
+/**
+ * converts image coordinates to earth coordinates.
+ * @param {Number} x x position of the point on the stage
+ * @param {Number} y y position of the point on the stage
+ * @returns {Array} latitude longitude of the point as a list.
+ */
+
+StaticMap.prototype.getEarthCoordinates = function(x, y){
+    return this._getMapInfo(this.socket.roleId).then(mapInfo => {
+        let coords = this._coordsAt(x,y, mapInfo);
+        return [coords.lat, coords.lon];
+    });
+};
+
+/**
+ * converts earth coordinates to image coordinates.
```

Comment:
This should be reworded to something like "Get the image coordinates (x, y) of a given location on the earth (latitude, longitude)"